.github/workflows/pr.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       run: gradle clean build
 
     - name: Run integration tests
-      run: gradle integrationTest
+      run: gradle integrationTest --info
```

Comment:
Ignore this, will remove before merging just wanted to be able to troubleshoot issues in CI more easily.

Reply:
Don't forget! :)